xapian-applications/omega/templates/activatelog

```diff
@@ -0,0 +1,2 @@
+$set{clicklog,true}
+$include{query}
```

Comment:
I checked the code, and it seems that currently this will actually work as-is.  However, we explicitly document it doesn't:

> Headers must be specified before any other
> output from the OmegaScript template - any `$httpheader{}`
> commands found later in the template will be silently ignored.

So it would be prudent not to rely on the current behaviour.